quickstart/inline-caching/cache-loader/src/main/java/io/vmware/event/ItemCacheLoader.java

```diff
@@ -0,0 +1,56 @@
+package io.vmware.event;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import org.jooq.DSLContext;
+import org.jooq.Record;
+import org.jooq.Result;
+import org.jooq.SQLDialect;
+import org.jooq.impl.DSL;
+
+import org.apache.geode.LogWriter;
+import org.apache.geode.cache.CacheFactory;
+import org.apache.geode.cache.CacheLoader;
+import org.apache.geode.cache.LoaderHelper;
+
+@SuppressWarnings({ "deprecation" })
+public class ItemCacheLoader implements CacheLoader<String, String> {
+	
+	private static LogWriter log;
+
+	static {
+		log = CacheFactory.getAnyInstance().getDistributedSystem().getLogWriter();
+	}
+
+	public String load(LoaderHelper helper) {
+		int itemId = Integer.parseInt((String) helper.getKey());
+		String userName = System.getProperty("postgres.username");
+		String password = System.getProperty("postgres.password");
+		String url = "jdbc:postgresql:postgres";
+		log.info("Cache miss... Loading data from postgres...");
+		log.info("itemId: "+itemId);
+		log.info("username: "+ userName);
+		log.info("password: "+ password);
+		try {
+			log.info("driver:" + Class.forName("org.postgresql.Driver"));
```

Comment:
Once again, why do the Class.forName call?

Reply:
Without it, the example doesn't work consistently. In older versions of the postgres driver this was documented as being necessary and for me, this allowed things to work. I'm curious if others have this issue as well. Look here for details: https://www.postgresql.org/docs/7.4/jdbc-use.html